gradle/libs.versions.toml

```diff
@@ -1,21 +1,19 @@
 [versions]
-managed-swagger = "2.2.7"
-# Required to keep catalog compatibility with 3.4.x.  Can be removed for 4.0.0
-managed-swagger-compat = "2.2.7"
+managed-swagger = "2.2.8"
```

Comment:
:x: It seems to be a mistake to keep `managed-swagger` in the versions, when the _library_ was removed below.

Reply:
@melix thanks for help! Now all works fine. Right solution is:
```groovy
micronautBom {
    suppressions {
        acceptedVersionRegressions.add("swagger-compat")
        acceptedLibraryRegressions.add("swagger")
    }
}
```
Need to remove library `swagger`, because right name now is `swagger-core`